src/Blocks/components/form/assets-admin/index.js

```diff
@@ -100,15 +94,14 @@ domReady(() => {
 	// Filter
 	////////////////////////////////////////////////////////////////
 
-	const selectorFilter = `.${adminListingManifest.componentJsFilterClass}`;
-	const elementsFilter = document.querySelector(selectorFilter);
+	const selectorFilter = getStateSelectorAdmin('listingFilter', true);
```

Comment:
Why create a variable if it is not necessary? 

I like to have this separate by feature so I will not be implementing these changes. Tnx for the suggestion.

Reply:
But you are using `getStateSelectorAdmin('listingItem', true);` on multiple places, so it would make sense to make it a variable?

Reply:
And why is the `const selectorFilter = getStateSelectorAdmin('listingFilter', true);` a variable then?

Reply:
> But you are using `getStateSelectorAdmin('listingItem', true);` on multiple places, so it would make sense to make it a variable?

agreed in most cases but in this case i chose not to follow this practice. In this instance `getStateSelectorAdmin` will only return value from object so there is no added benefit on using variable regarding optimizations (memory, speed, etc). If you open the file in full you will see that the function are separated by features, so reusing this as a variable would add unecesery coupling of features and make later refactoring harder. By using this approach, I have everything decoupled.

For the `const selectorFilter = getStateSelectorAdmin('listingFilter', true);` this is in the variable because I'm it in multiple places in **one** feature.

Reply:
Ok, didn't make sense on a first glance, thanks for explaining 👍